src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
Якщо батьківский елемент переіменувати на navigation__list, то тут теж треба підправити
Слідуємо BEM block__element-name - можна вказати назву елементу через дефіс navigation__list-item або не дублювати весь батьківський селектор, а залишити просто navigation__item, я б обрала другий варіант
```suggestion
          <li class="navigation__list-item">
          або
          <li class="navigation__item">
```